src/main/java/frc/robot/utils/VisionUtils.java

```diff
@@ -24,17 +43,16 @@ public static Pair<Pose2d, Double> getVisionPose() {
         if (hasTarget == 0)
             return null;
 
-        double[] limeLightPose = VisionConstants.LIMELIGHT_TABLE.getEntry("botpose").getDoubleArray(new double[0]);
-        if (limeLightPose.length != 6)
+        double[] robotPose = VisionConstants.LIMELIGHT_TABLE.getEntry("botpose").getDoubleArray(new double[0]);
+        if (robotPose.length != 6)
             return null;
 
         double latency = VisionConstants.LIMELIGHT_TABLE.getEntry("tl").getDouble(0);
-        limeLightPose[0] = Constants.FIELD_WIDTH / 2 - limeLightPose[0];
-        limeLightPose[1] = limeLightPose[1] + Constants.FIELD_HEIGHT / 2;
+        robotPose[0] = Constants.FIELD_WIDTH / 2 + robotPose[0];
+        robotPose[1] = robotPose[1] + Constants.FIELD_HEIGHT / 2;
 
-        Rotation2d robotRotation = Rotation2d.fromDegrees(limeLightPose[5]);
-        Translation2d robotTranslation = new Translation2d(limeLightPose[0], limeLightPose[1])
-                .minus(VisionConstants.CAMERA_OFFSET.rotateBy(robotRotation));
+        Rotation2d robotRotation = Rotation2d.fromDegrees(robotPose[5]);
+        Translation2d robotTranslation = new Translation2d(robotPose[0], robotPose[1]);
```

Comment:
what's robot translation?
naming is important!

Reply:
the default of any vector if not specified is to the origin that's the translation from the origin